docs/output.md

```diff
@@ -168,7 +168,7 @@ newly recovered components are stored in a file (components_recovered.csv).
 - `pixelator`
 
   - `annotate`
-    - `<sample-id>.dataset.pxl`
+    - `<sample-id>.annotated.dataset.pxl`
```

Comment:
annotated -> annotate

Let's use the stage name instead of a participle.

Reply:
Your right, it should be annotate.